modules/persistent-storage-csi-migration-sc.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-migration-sc-implications_{context}"]
+= Storage class implications
+
+For new {product-title} 4.11, and later, installations, the default storage class is the CSI storage class. All volumes provisioned using this storage class are CSI persistent volumes (PVs).
```

Comment:
That's not true for all drivers, i am not sure if a generic statement works here. cc @gnufied

Reply:
Yeah I don't think that statement works or is true.  Either we have to create a table that explicitly lists in which release migration for that particular storage driver went GA  or we shouldn't mention this at all.

Reply:
yes that would work too. There are some late changes on the vsphere side so I would hold on for the moment

Reply:
Done.